package.json

```diff
@@ -55,27 +55,27 @@
   "dependencies": {
     "eventemitter3": "^5.0.1",
     "p-defer": "^4.0.1",
-    "rpc-reflector": "^3.0.0"
+    "rpc-reflector": "^3.0.1"
   },
   "peerDependencies": {
-    "@comapeo/core": "^4.1.3"
+    "@comapeo/core": "^4.3.0"
```

Comment:
Maybe being dramatic, but I think this technically could be a breaking change? I think we're using 4.3.0 everywhere it matters so not overly worried about how it affects us, but open to reverting this if desired.

Reply:
I think it's fine.